test/sanitizer_suppressions/ubsan

```diff
@@ -62,6 +63,7 @@ implicit-integer-sign-change:script/bitcoinconsensus.cpp
 implicit-integer-sign-change:script/interpreter.cpp
 implicit-integer-sign-change:serialize.h
 implicit-integer-sign-change:txmempool.cpp
+implicit-integer-sign-change:txmempool_entry.h
```

Comment:
Are they still needed?

Reply:
Apparently, they do not. Removed in needless #26542.

Thank you!